packages/jellyfish-wallet-classic/src/classic.ts

```diff
@@ -0,0 +1,65 @@
+import { WalletEllipticPair } from '@defichain/jellyfish-wallet'
+import { EllipticPair } from '@defichain/jellyfish-crypto'
+import {
+  Transaction,
+  Vout,
+  TransactionSegWit
+} from '@defichain/jellyfish-transaction'
+import { SignInputOption, TransactionSigner } from '@defichain/jellyfish-transaction-signature'
+
+/**
+ * WalletClassic extends WalletEllipticPair with a simple classic implementation.
+ *
+ * Single elliptic pair wallet.
+ */
+export class WalletClassic implements WalletEllipticPair {
```

Comment:
Possibly, in the future we can include these static helper method to make it easier to create Classic Instance

1. `WalletClassic.fromWIF(string)`
2. `WalletClassic.fromPrivkey(Buffer)`